README.md

```diff
@@ -50,3 +50,38 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Operations
+
+The backend is now in Trello. The local .env file contains the variables. These are not shared in Github
+
+Items are marked To Do, Doing and Done. The 'complete action' button has been added to move any item to Done *
+
+
+* Future revision should limit this to To Do and Doing Only.
+
+## Working with pytest
+
+poetry install pytest
+
+To run a test session for unit testing :-
+
+poetry run pytest
+
+## Working with GIT
+
+git add --all
+
+git commit -m "<add change details>"
+  
+git push origin module-2
+
+## Working with Selenium
+
+poetry add selenium
+
+selenium --version
+
+
+  
```

Comment:
You should mention how to run the tests (`poetry run pytest`) and any requirements, (like Firefox & geckodriver)

Reply:
done